Makefile

```diff
@@ -112,23 +121,49 @@ ifeq ($(TARGET_ARCH),arm64)
 endif
 endif
 
+ifeq ($(TARGET_OS),windows)
+ifeq ($(SOURCE_OS),linux)
+ifeq ($(TARGET_ARCH),amd64)
+    GO_TAGS ?= -tags no_cgo
+    X264_ROOT ?= $(shell pwd)/x264/windows-amd64
```

Comment:
what's the difference between using `shell pwd` vs. `.` relative paths? Do we need absolute in this case?

Reply:
I generally like using absolute paths, relative would probably also work here.